### PR TITLE
Clarify documentation for using ingress-dns plugin on Linux with Network Manager

### PR DESCRIPTION
**Pull request by oprudkyi**
Details for ingress-dns setup on Linux with Network Manager
verified on 
openSUSE Leap 15.3
NetworkManager 1.22.10
Dnsmasq version 2.86

### CODE REVIEW

**Comment by oprudkyi** [comment]
I signed it

File: site/content/en/docs/handbook/addons/ingress-dns.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ minikube addons enable ingress-dns
 {{% tabs %}}
 {{% linuxtab %}}
 
+## Linux OS with resolvconf
```
Comment (medyagh) [inline site/content/en/docs/handbook/addons/ingress-dns.md:64]: how about we create a "nested Tab "
```
{{% tabs %}}
{{% withresolveconf %}}...

...
{{% withNetworkManager %}}...
{{% /tabs %}}

Comment (oprudkyi) [reply site/content/en/docs/handbook/addons/ingress-dns.md:64]: Hi @medyagh 
it seems like hugo/docsy etc doesn't support embeding tabs into tabs 
image

**Review by sharifelgamal — APPROVED** [review]

**Comment by sharifelgamal** [comment]
Thanks for contributing to minikube documentation!

**Pull request merged**